signer/src/bitcoin/packaging.rs

```diff
@@ -603,6 +596,28 @@ mod tests {
             Self { votes, ..Default::default() }
         }
 
+        /// Create a new request item with random votes against.
+        fn with_rng(rng: &mut impl Rng) -> Self {
+            let mut votes = [false; 5];
+            for vote in &mut votes {
+                *vote = rng.gen_bool(0.5);
+            }
+            let needs_signature = rng.gen_bool(0.5);
+            let vsize = rng.gen_range(1..=10000);
+            let withdrawal_id = if needs_signature && rng.gen_bool(0.5) {
```

Comment:
I think you want this to be just `if !needs_signature { ...`, because deposits need a signature and withdrawals do not.

Reply:
[b4a5ef5](https://github.com/stacks-sbtc/sbtc/pull/1664/commits/b4a5ef585e6562ac04c459800e2f9631e157bf57)